platform/core/src/defaults/hotkeyBindings.js

```diff
@@ -0,0 +1,95 @@
+export default [
+  // ~ Global
+  {
+    commandName: 'incrementActiveViewport',
+    label: 'Next Viewport',
+    keys: ['right'],
+  },
+  {
+    commandName: 'decrementActiveViewport',
+    label: 'Previous Viewport',
+    keys: ['left'],
+  },
+  // Supported Keys: https://craig.is/killing/mice
+  // ~ Cornerstone Extension
+  { commandName: 'rotateViewportCW', label: 'Rotate Right', keys: ['r'] },
+  { commandName: 'rotateViewportCCW', label: 'Rotate Left', keys: ['l'] },
+  { commandName: 'invertViewport', label: 'Invert', keys: ['i'] },
+  {
+    commandName: 'flipViewportVertical',
+    label: 'Flip Horizontally',
+    keys: ['h'],
+  },
+  {
+    commandName: 'flipViewportHorizontal',
+    label: 'Flip Vertically',
+    keys: ['v'],
+  },
+  { commandName: 'scaleUpViewport', label: 'Zoom In', keys: ['+'] },
+  { commandName: 'scaleDownViewport', label: 'Zoom Out', keys: ['-'] },
+  { commandName: 'fitViewportToWindow', label: 'Zoom to Fit', keys: ['='] },
+  { commandName: 'resetViewport', label: 'Reset', keys: ['space'] },
+  // clearAnnotations
+  { commandName: 'nextImage', label: 'Next Image', keys: ['down'] },
+  { commandName: 'previousImage', label: 'Previous Image', keys: ['up'] },
+  // firstImage
+  // lastImage
+  {
+    commandName: 'previousViewportDisplaySet',
+    label: 'Previous Series',
+    keys: ['pagedown'],
+  },
+  {
+    commandName: 'nextViewportDisplaySet',
+    label: 'Next Series',
+    keys: ['pageup'],
+  },
+  // ~ Cornerstone Tools
+  { commandName: 'setZoomTool', label: 'Zoom', keys: ['z'] },
+  // ~ Window level presets
+  {
+    commandName: 'windowLevelPreset1',
+    label: 'W/L Preset 1',
+    keys: ['1'],
+  },
+  {
+    commandName: 'windowLevelPreset2',
+    label: 'W/L Preset 2',
+    keys: ['2'],
+  },
```

Comment:
These do not have default values/options. Can we hardcode options for the values we have set in the wwwc dropdown?

Reply:
@dannyrb since hotkeysmanager uses the commandName as the key for the map of bindings, we cant just use one command and then pass the options in each binding, maybe i should update the manager to use a different property as key? currently each commandname has its own values/options like windowlevel1, windowlevel2, etc..

Reply:
It's okay if we have to tackle it that way for now. If there's an easy way to add the functionality from this stale/closed PR, we could also do that. It was drafted to address this problem, but was not merged:

https://github.com/OHIF/Viewers/pull/1456/files